thrift_includes.h

```diff
@@ -0,0 +1,25 @@
+#ifdef P4THRIFT
```

Comment:
I think this is bad style to have a file like this in the root directory and to include unconditionally everything Thrift-related (i.e. in some cases you include more than you need). I think the best solution is still to have a `#ifdef P4THRIFT` switch in each source file that requires some Thrift headers, and only include what you need.
There would be another solution requiring to have autoconf process the file, but I think that would be an overkill at the point and would make the source code more complicated.